utoipa/src/lib.rs

```diff
@@ -1020,22 +1020,22 @@ mod tests {
             (
                 "u8",
                 u8::schema(),
-                json!({"type": "integer", "format": "uint8", "minimum": 0.0}),
+                json!({"type": "integer", "format": "uint8", "minimum": 0}),
```

Comment:
This appears to be caused by https://github.com/juhaku/utoipa/pull/618 - which is great.
but I am confused why the `test_partial_schema_strict_integers` test directly above this one doesnt also need the `.0` removed..?

Reply:
It was forgotten from previous commit to update this test. The thing seemingly is because the number comparison is done differently. This from the test above.
```rust
let config = Config::new(CompareMode::Strict).numeric_mode(NumericMode::AssumeFloat);
assert_json_matches!(schema, value, config);
```

Reply:
Anyhow I think we can live with that difference, and merge this to the master :smile: